src/systolic_array/commit.cpp

```diff
@@ -163,14 +167,14 @@ void Commit::localSpadCallback(PacketPtr pkt) {
       // If the outputs are finished, do the activation function before we send
       // the outputs back to the scratchpad.
       activationFunc(lineSlotPtr->getDataPtr<uint8_t>(),
-                     elemsPerLine,
+                     pkt->getSize() / accel.elemSize,
```

Comment:
If pkt size is not a multiple of accel.elemSize, what happens? Is that possible? If not, please add an assertion or fatal warning to enforce it.

Reply:
The pkt size will always be a multiple of accel.elemSize. Added the assert.